packages/mui-joy/src/Textarea/Textarea.tsx

```diff
@@ -66,10 +66,10 @@ const TextareaRoot = styled('div', {
         '--Icon-fontSize': '1.75rem',
       }),
       // variables for controlling child components
-      '--internal-paddingBlock':
+      '--_Textarea-paddingBlock':
```

Comment:
Out of curiosity why is this private?

Reply:
Currently, the `_Textarea-paddingBlock` is used to calculate other things (as if it is an intermediate variable), e.g. radius of the children. It is not intended for customization, so I keep it as private at this point.

For the time being, if I am not sure of its usage I will keep these kinds of variables private to implicitly tell developers that "don't alter them because they are private".